### ISSUE TITLE
Installing 8.8.rc0 does now upgrade sage-brial

### ISSUE THREAD

**Issue by strogdon** [cschwan/sage-on-gentoo#540]
`emerge -C sage && emerge -1 sage` did not upgrade `sage-brial`

```
[U] dev-python/sage-brial [1]
     Available versions:  (~)1.2.5^m
     Installed versions:  1.2.4^m(08:44:45 PM 12/06/2018)(PYTHON_TARGETS="python2_7 python3_6 -python3_4 -python3_5")
     Homepage:            https://github.com/BRiAl/BRiAl
     Description:         BriAL, a successor to PolyBoRI: Polynomials over Boolean Rings

[I] sci-libs/brial [1]
     Available versions:  (~)1.2.5^m {doc static-libs}
     Installed versions:  1.2.5^m(06:03:51 PM 06/09/2019)(-doc -static-libs)
     Homepage:            https://github.com/BRiAl/BRiAl
     Description:         BriAL, a successor to PolyBoRI: Polynomials over Boolean Rings

[1] "sage-on-gentoo" /var/lib/layman/sage-on-gentoo
```
This is not in Prefix.

**Comment by kiwifb** [comment]
That's not strictly necessary. I forced brial because it was important for gcc 9.1.0. At some point I wanted to make sage-brial its own package upstream - which would have had its own numbering scheme. There was opposition to that in debian and some in sage itself.

But really there has been no changes in sage-brial python code in this, and possibly previous, release(s).

**Comment by kiwifb** [comment]
I know it is a bit ugly and there is no real policy on forcing package upgrade. I often find old versions of packages that have been removed from the tree in my box when I run `depclean`. And they are there because their dependencies have not forced an upgrade they did not deem necessary.

**Comment by strogdon** [comment]
I have seen this also. For example
```
eix giac

[U] sci-mathematics/giac
     Available versions:  ~1.1.0[2] ~1.2.3.57[1] (~)1.4.9.45-r1[1] (~)1.4.9.71-r1[1] (~)1.5.0.59[1] {ao doc +ecm examples fltk gc +glpk l10n_ static-libs L10N="el en es fr pt"}
     Installed versions:  1.5.0.53[1](09:05:01 PM 05/18/2019)(ecm glpk -ao -doc -examples -fltk -gc -static-libs L10N="en fr -el -es -pt")
     Homepage:            http://www-fourier.ujf-grenoble.fr/~parisse/giac.html
     Description:         A free C++ CAS (Computer Algebra System) library and its interfaces

[1] "sage-on-gentoo" /var/lib/layman/sage-on-gentoo
```
and thought `sage-brial` may be similar. Non-issue.
